src/shared/hwdb-util.c

```diff
@@ -501,43 +501,39 @@ static int import_file(struct trie *trie, const char *filename, uint16_t file_pr
                                 break;
 
                         if (line[0] == ' ') {
-                                r = log_syntax(NULL, LOG_WARNING, filename, line_number, SYNTHETIC_ERRNO(EINVAL),
-                                               "Match expected but got indented property \"%s\", ignoring line.", line);
+                                log_syntax(NULL, LOG_WARNING, filename, line_number, 0,
+                                           "Match expected but got indented property \"%s\", ignoring line.", line);
```

Comment:
hmm, you use `return r` at the end of the function, and suddenly this will always be zero, is that intended?

Reply:
It had always been zero before. In the loop we store the result of read_line in `r`.

Reply:
Right, the value assigned to `r` here was never used, and overwritten by the next call of `read_line_full()`.